src/Chainweb/Rosetta/Utils.hs

```diff
@@ -762,6 +762,8 @@ getCmdPayload (Command p _ _) =
     (decodeStrict' $! T.encodeUtf8 p)
 
 
+-- TODO: This assumes Rosettas signatures are all Ed25519 signatures
+-- (Not webauthn).
```

Comment:
Per Linda: we're not looking to continue to support Rosetta, so please feel free to do what you need to in order to make rosetta work here.